back/benchmarks/Main.hs

```diff
@@ -0,0 +1,49 @@
+-- | Module contains all stuff to migrate from AcidState to Postgres.
+module Main
+       (
+         main
+       ) where
+
+import Imports
+
+import Gauge
+import Hasql.Transaction.Sessions (Mode (..))
+
+import Guide.Database.Queries.Update
+import Guide.Database.Queries.Select
+import Guide.Types.Core
+import Guide.Database.Connection
+
+
+main :: IO ()
+main = do
+    conn <- connect
+    defaultMain [databaseBenchmark conn]
+  where
+    databaseBenchmark conn =
+      bgroup
+        "Database"
+        [ bench "select" $ nfIO $
+            runTransactionExceptT conn Read $ selectCategory "category1111"
+        , bench "updete" $ nfIO $
+            runTransactionExceptT conn Write $ updateCategory "category1111" update
+        ]
+    update :: Category -> Category
+    update = _categoryTitle .~ "title10"
+
+{-
+benchmarked Database/select
+time                 496.1 μs   (429.1 μs .. 551.4 μs)
+                     0.932 R²   (0.868 R² .. 0.976 R²)
+mean                 590.7 μs   (502.5 μs .. 939.6 μs)
+std dev              508.6 μs   (51.75 μs .. 1.065 ms)
+variance introduced by outliers: 97% (severely inflated)
```

Comment:
You should run benchmarks without anything running in background (browsers, etc), otherwise they will be imprecise. "severely inflated" is not good.

Reply:
I reduced inflated parameter to 53% and 39%